webserver_config.py

```diff
@@ -0,0 +1 @@
+AUTH_ROLE_PUBLIC = 'Admin'
```

Comment:
is this file needed?

Reply:
ah it's to disable web login page not necessary but i did have it with airflow bridge